_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,31 @@
+## Node hosts
+
+- AMD64 processor
+
+- Linux kernel 3.10 or later. The following distributions are known to work 
+  well with {{site.prodname}} and {{include.orch}}.
+  - RedHat Linux 7{% if include.orch == "Kubernetes" or include.orch == "host protection" %}
+  - CentOS 7
+  - CoreOS Container Linux stable
+  - Ubuntu 16.04
+  - Debian 8
+  {% endif %}{% if include.orch == "OpenShift" %}
+  - CentOS 7
+  {% endif %}{% if include.orch == "OpenStack" %}
+  - Ubuntu 16.04
+  - CentOS 7
+  {% endif %}
+{% if include.orch == "Kubernetes" %}
+<br>
+- No other pod networking configurations stored in `/etc/cni/net.d`.
+{% endif %}
+
+## Key/value store
+
+{{site.prodname}} {{page.version}} requires an [etcdv3 or later server](https://github.com/coreos/etcd/releases) accessible by all nodes in the cluster. 
```

Comment:
This first line is wrong

Reply:
How about something like this:
-------------------------------------------------------

## Key/Value store

Calico requires a key/value store accessible by all Calico components. Calico supports one of the following options for its data store:

- etcdv3
- Kubernetes API: If using Kubernetes, Calico can use the Kubernetes API for data storage using custom resource definitions.